R/format_PEW_temp.R

```diff
@@ -0,0 +1,541 @@
+
+  Brood_data_sel <-
+    Brood_data %>%
+    dplyr::select(BreedingSeason, Species, PopID, BroodID, LocationID,
+                  HatchDate_observed)
+
+
+  Capture_data_temp <-
+    pew_data %>%
+    #### Rename variables
+    dplyr::rename(LocationID = NestboxID) %>%
+    dplyr::select(-DateRingingChicks) %>%
+    #### Create new variables
+    dplyr::mutate(#### ASSUMPTION: There is no captureID info, so I made this up
+                  CaptureID = paste(IndvID, LocationID,
+                                    paste0(lubridate::year(CaptureDate),
+                                           sprintf("%02d", lubridate::month(CaptureDate)),
+                                           sprintf("%02d", lubridate::day(CaptureDate))),
+                                    sep = "_"),
+      #### ASSUMPTION: Some times are strange, to simplify, round to whole hour
+      #### ? do they correspond only to the experimental observations? keep/remove?
+      CaptureTime = ifelse(is.na(ObservationTimeH),
+                           NA,
+                           paste(sprintf("%02d", round(ObservationTimeH, 0)),
+                                 "00", sep = ":")),
+      Sex_observed = ifelse(Sex == "Chick", NA_character_, substr(Sex, 1, 1)),
+      CaptureAlive = ifelse(Method != "Found dead", TRUE, FALSE),
+      #### ASSUMPTION: the same as CaptureAlive, as there is no additional data
+      ReleaseAlive = CaptureAlive,
+      CapturePopID = PopID,
+      CapturePlot = NA_character_,
+      ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+      ReleasePlot = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+      #### ASSUMPTION: use NA, as there is no information
+      OriginalTarsusMethod = NA_character_,
+      WingLength = NA,
+      ExperimentID = dplyr::case_when(Experiment == "BSM - Griffioen et al. 2019 PeerJ" ~
+                                        "COHORT; PARENTAGE",
+                                      Experiment == "2h Temp D4" ~ "SURVIVAL",
+                                      Experiment == "2h Temp D4 + Handicaping Male: Griffioen et al. 2019 Front Ecol&Evol" ~
+                                        "SURVIVAL; PARENTAGE"),
+      #### NOT SURE ABOUT THIS
+      Age_observed = case_when(Age == "0" & Method == "ChickRinging" ~ 1,
+                               Age == "1" ~ 5,
+                               Age == ">=2" ~ 7,
+                               #### Account for controls in December
+                               Age == "1" & Method == "Night Control Winter" &
+                                 lubridate::month(CaptureDate) == 12 ~ 6,
+                               #### Account for controls in January & February
+                               Age == ">=2" & Method == "Night Control Winter" &
+                                 lubridate::month(CaptureDate) %in% c(1, 2) ~ 8,
+                               is.na(Age) ~ 5),
+      Age_observed = as.integer(Age_observed))
```

Comment:
Can explicitly make a number an integer with `L`

```suggestion
      Age_observed = case_when(Age == "0" & Method == "ChickRinging" ~ 1L,
                               Age == "1" ~ 5L,
                               Age == ">=2" ~ 7L,
                               #### Account for controls in December
                               Age == "1" & Method == "Night Control Winter" &
                                 lubridate::month(CaptureDate) == 12 ~ 6L,
                               #### Account for controls in January & February
                               Age == ">=2" & Method == "Night Control Winter" &
                                 lubridate::month(CaptureDate) %in% c(1, 2) ~ 8L,
                               is.na(Age) ~ 5L))
```

Reply:
Ok.